modules/llama_cpp_plugin/notebooks/llama_cpp_plugin_with_qwen.ipynb

```diff
@@ -0,0 +1,276 @@
+    "如果一个问题没有任何意义或与事实不符，请解释原因，而不是回答错误的问题。如果您不知道问题的答案，请不要分享虚假信息。另外，答案请使用中文。\\\n",
+    "\"\"\"\n",
+    "\n",
+    "model_configuration = {\n",
+    "    \"model_id\": \"Qwen/Qwen-7B-Chat\",\n",
+    "    \"remote\": True,\n",
+    "    \"start_message\": f\"<|im_start|>system\\n {DEFAULT_SYSTEM_PROMPT_CHINESE }<|im_end|>\",\n",
+    "    \"history_template\": \"<|im_start|>user\\n{user}<im_end><|im_start|>assistant\\n{assistant}<|im_end|>\",\n",
+    "    \"current_message_template\": '\"<|im_start|>user\\n{user}<im_end><|im_start|>assistant\\n{assistant}',\n",
+    "    \"stop_tokens\": [\"<|im_end|>\", \"<|endoftext|>\"]\n",
+    "}\n",
+    "\n",
+    "model_name = model_configuration[\"model_id\"]\n",
+    "start_message = model_configuration[\"start_message\"]\n",
+    "history_template = model_configuration.get(\"history_template\")\n",
+    "current_message_template = model_configuration.get(\"current_message_template\")\n",
+    "stop_tokens = model_configuration.get(\"stop_tokens\")\n",
+    "tokenizer_kwargs = model_configuration.get(\"tokenizer_kwargs\", {})\n",
+    "\n",
+    "def convert_history(history):\n",
+    "    text = start_message + \"\".join([\"\".join([history_template.format(num=round, user=item[0], assistant=item[1])]) \n",
+    "                                    for round, item in enumerate(history[:-1])])\n",
+    "    text += \"\".join([\"\".join([current_message_template.format(num=len(history) + 1, user=history[-1][0], assistant=history[-1][1])])])\n",
+    "    return text"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2b543737-eb59-453c-99f5-236fa162ada3",
+   "metadata": {},
+   "source": [
+    "Now we can build the input tokens representing the user prompt to the model. The prompt is adjusted to conform to the expected chatbot template using the utility functions defined earlier and then tokenized using the corresponding tokenizer.\n",
+    "\n",
+    "**Note:** Although the GGUF file contains the vocabulary and the tokenizer information, the tokenization and detokenization steps are currently not part of the `LLAMA_CPP` flow. These steps of the LLM processing pipeline should be executed using other means (e.g. in Python you can instantiate the required tokenizers from the `transformers` library if these are available for your model, and optionally convert these to OpenVINO representation separately to be executed with other plugins such as 'CPU')."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "41f70a67-729c-4f3b-810c-d273d455e49e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "user_prompt = \"孙悟空是谁?\"\n",
+    "# user_prompt = \"太阳为什么是黄色的?\"\n",
+    "\n",
+    "formatted_input_prompt = convert_history([[user_prompt, \"\"]])\n",
+    "\n",
+    "from transformers import AutoTokenizer\n",
+    "tok = AutoTokenizer.from_pretrained(model_name, trust_remote_code=True)\n",
```

Comment:
`tok` -> `tokenizer`, just like it is used in HF samples.

Reply:
Done